spec/src/main/asciidoc/chapters/applications.asciidoc

```diff
@@ -8,26 +8,26 @@ This chapter introduces the notion of an MVC application and explains how it rel
 MVC Applications
 ~~~~~~~~~~~~~~~~
 
-An MVC application consists of one or more JAX-RS resources that are annotated with `@Controller` and, just like JAX-RS applications, zero or more providers. If no
-resources are annotated with `@Controller`, then the resulting application is a JAX-RS application instead. In general, everything that applies to a JAX-RS application
-also applies to an MVC application. Some MVC applications may be _hybrid_ and include a mix of MVC controllers and JAX-RS resource methods.
+An MVC application consists of one or more JAX-RS resources that are annotated with `@Controller` and, just like JAX-RS applications, zero or more providers.
```

Comment:
Shouldn't it be "**A** MVC application"?

Reply:
To be honest, I'm not completely sure. I also thought "A MVC application" would be correct, but languagetool.org told me that "An MVC application" is the correct version.

Maybe because MVC is an abbreviation the next word is "application" which starts with a vowel!?!

Reply:
OK, let's leave it like it is. Thanks!

Reply:
An abbreviation starting with M (which is pronounced 'em') is treated as a word starting with a vowel...